src/Sulu/Bundle/MediaBundle/Resources/js/components/MimeTypeIndicator/MimeTypeIndicator.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React from 'react';
+import {Icon} from 'sulu-admin-bundle/components';
+import MimeTypeMapper from './MimeTypeMapper';
+import mimeTypeIndicatorStyles from './mimeTypeIndicator.scss';
+
+type Props = {
+    width?: number,
+    height?: number,
+    iconSize?: number,
```

Comment:
Do we need different icon sizes here?

Reply:
yes, an indicator for a `25x25` image needs a smaller icon than one for a `400x400` thumbnail